dateparser/utils.py

```diff
@@ -86,3 +86,24 @@ def find_date_separator(format):
     m = re.search(r'(?:(?:%[dbBmaA])(\W))+', format)
     if m:
         return m.group(1)
+
+
+class Registry(type):
```

Comment:
It's not very encouraged to use metaclasses on libraries as we can never know what our users can do with it. They may want to subclass it with their own metaclass.